src/main/java/HWS/HW10/MessageGenerator.java

```diff
@@ -0,0 +1,29 @@
+package HWS.HW10;
+
+public class MessageGenerator {
+
+    public static void main(String[] args) {
+        String info = MessageGenerator.infoMessage("Запуск додатку", "Версія 1.0");
+        System.out.println(info);
+        String error = MessageGenerator.errorMessage("Файл не знайдено", 404);
+        System.out.println(error);
+        String loginToComfy  = MessageGenerator.successLogin("Comfy");
+        System.out.println(loginToComfy);
+    }
```

Comment:
No need in extra code
```suggestion
        String info = infoMessage("Запуск додатку", "Версія 1.0");
        System.out.println(info);
        String error = errorMessage("Файл не знайдено", 404);
        System.out.println(error);
        String loginToComfy  = successLogin("Comfy");
        System.out.println(loginToComfy);
    }
```